demos/multi_channel/face_detection_demo/main.cpp

```diff
@@ -313,7 +313,7 @@ int main(int argc, char* argv[]) {
             }
 
 
-            std::vector<Detections> detections(getTensorHeight(output->getTensorDesc()) / 200);
+            std::vector<Detections> detections(batchSize);
```

Comment:
The function has a way to extract batch size, so there is no need to introduce an extra argument. The way the function extracts the value is weird, but this is different problem why dldt extends height instead of batch dimension, so introducing `batchSize` argument should be reverted.